tcms/testruns/static/testruns/js/get.js

```diff
@@ -277,6 +277,15 @@ function fileBugFromExecution(execution) {
         const trackerId = $('.one-click-bug-report-form #id-issue-tracker').val()
         jsonRPC('Bug.report', [execution.id, trackerId], result => {
 
+            const testExecutionRow = $(`.test-execution-${execution.id}`)
+            animate(testExecutionRow, () => {
+                testExecutionRow.find('.test-execution-hyperlinks').append(renderLink({ is_defect: true, url: result.response }))
```

Comment:
See the original tesrun_actions.js lines 408-413 for unescaping & handling errors.